### ISSUE TITLE
Pattern view settings

### ISSUE THREAD

**Issue by WallK** [embroidery-space/embroidery-studio#56]
Acceptance Criteria

    Main interface should have a view selector with following options:
        Pixels (color squares?)
        Stitch preview
        Symbols

**Comment by niusia-ua** [comment]
I considered the following options:

- `Solid` — a simple stitch representation.
  In this view, the squares are primarily used.
- `Stitches` — still simple but some stitches (e.q. part stitches) have more complex and real-like shapes.
- `Symbols` — in this case the font symbols are used.
  It is also possible to combine it with the `Solid` view to colourize the image.
- `Preview` — this view will use the pre-made stitch images to simulate the real embroidery.

What do you mean about these options?
I can provide some assets to clearly show the views.

**Comment by WallK** [comment]
This is a good list of options!
I think "combining with solid" could supersede the Background option in Symbol settings too
As in:
Symbols as set up
Force black symbols on colored backgrounds
Force black symbols on transparent background

Also, preview thing could be a part of stitches
With option for "flat" or "textured"

**Comment by niusia-ua** [comment]
> I think "combining with solid" could supersede the Background option in Symbol settings too

Actually, no. "Combining" means that there is more than one option to combine with. I said "combine" just to illustrate the result. As for me, it is better to allow the user to select what colour he wants to use:
- without background colour at all
- use the floss colour
- use the custom colour

> With option for "flat" or "textured"

Can you clarify what "flat" and "textured" previews mean?

**Comment by WallK** [comment]
> As for me, it is better to allow the user to select what colour he wants to use:
    without background colour at all
    use the floss colour
    use the custom colour
Isn't this exactly what is written in https://github.com/embroidery-space/embroidery-studio/issues/54 ?

>Can you clarify what "flat" and "textured" previews mean?
Flat is solid color
Textured -- more 3D, lifelike. The same idea you have with "premade stitches"

**Comment by niusia-ua** [comment]
> Isn't this exactly what is written in #54 ?

Yes, that is exactly what I mean. Just to do not misunderstand each other.
